pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lit-llama"
+version = "0.1.0"
+description = "Implementation of the LLaMA language model"
+license = {text = "Apache-2.0"}
+authors = [
+    { name = "Lightning AI", email = "community@lightning.ai" }
+]
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
In LitGPT we require >= 3.10 because we used some features not supported in 3.9 (I don't remember which ones); so maybe also make this 3.10 here to be on the safe side

Reply:
```suggestion
requires-python = ">=3.10"
```